blocks/modal/modal.css

```diff
@@ -0,0 +1,40 @@
+.modal dialog {
+  --dialog-border-radius: 16px;
+
+  overscroll-behavior: none;
+  padding: 30px;
+  border: 1px solid #ccc;
+  border-radius: var(--dialog-border-radius);
+  left: 20px;
+  right: 20px;
+  width: clamp(300px, 80%, 700px);
```

Comment:
i think the clamp provides an odd experience on mobile (close button is cut off)

Reply:
fixed now for ios and android